lib/3scale/tasks/helpers/command.rb

```diff
@@ -0,0 +1,65 @@
+module ThreeScale
+  module Tasks
+    module Helpers
+      module Command
```

Comment:
If this module is not used, I'd rather not include it.

Reply:
This is a byproduct of this work - the issue is how likely are you to have a use for it in the future?

In fact, there is a similar `run_command` method elsewhere in the tasks, but that case was solved with just `system`, because there was no need to capture output. This is likely to be useful in the future - not including it means losing it while having good chances someone will have to solve the same issue again.

Reply:
IMO code that is not used should not be included. 

If it is very likely to be used in near future, unmerged PR with the method would be appropriate.

Reply:
Ok, I will remove this from the PR.